src/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs

```diff
@@ -32,19 +30,136 @@ public void CheckSourceExistenceAndDeletion()
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
consider replacing all these nano server ones with
`[ConditionalFact(typeof(Helpers), nameof(Helpers.SupportsEventLogs))]`
so they don't have to concern themselves with what Nano server is and whether anything else supports event logs.